### PR TITLE
add b == 6

### PR DESCRIPTION
**Pull request by GYHX**

### CODE REVIEW

File: shim/main.rs [hunk 1]
```diff
@@ -1,3 +1,4 @@
 fn main() {
     let mut a = 5;
+    let b = 6;
```
Comment (yaohuixguo) [inline shim/main.rs:3]: why add b =6

Comment (GYHX) [reply shim/main.rs:3]: this is test

Comment (yaohuixguo) [reply shim/main.rs:3]: you just add b

Comment (yaohuixguo) [reply shim/main.rs:3]: why resolve conversation

**Pull request closed without merging**